### ISSUE TITLE
schema aware tests

### ISSUE THREAD

**Issue by benibela** [w3c/qt3tests#34]
fo-test-fn-id-001 fo-test-fn-id-002 fo-test-fn-element-with-id-001 fo-test-fn-element-with-id-002 fo-test-fn-idref-001 fo-test-fn-idref-002 have a `validate lax {`  expression, but no dependency on schema validation and do not allow XQST0075 as output

**Comment by michaelhkay** [comment]
I had trouble working out which tests you are referring to here - eventually found them in app-spec-examples. These tests are generated from the example in the spec and the markup in the spec isn't enough to generate the dependency information; I guess it could be added by hand now that the spec is stable.

**Comment by nkutsche** [comment]
Hi @michaelhkay, 

I ran into a similar problem. I'm trying to implement a custom test-runner based on XSpec & Saxon-HE.

I have now found the test cases for `fn:element-with-id` (see [here](https://github.com/w3c/qt3tests/blob/master/fn/element-with-id.xml)). As far as I understand these test cases are also schema-aware and (currently) the only indicator that they won't work with Saxon-HE would be that `environment/source/@validation = 'strict'` is set, correct?

**Comment by michaelhkay** [comment]
Yes, validation=strict in this case is the only indication that a schema-aware processor is needed. (Technically according to the XQuery spec it's a dependency on the "typed data feature", but in Saxon, that feature comes with schema-awareness).
